### ISSUE TITLE
A11y - OU: Links must have discernible text

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5469]
Ensure links have discernible text. [https://dequeuniversity.com/rules/axe/4.10/link-name?application=AxeChrome|https://dequeuniversity.com/rules/axe/4.10/link-name?application=AxeChrome|smart-link] 

h3. Element Location:

{noformat}.b2b-create-back{noformat}

h3. To solve this problem, you need to fix at least (1) of the following:

* Element does not have text that is visible to screen readers
* aria-label attribute does not exist or is empty
* aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
* Element has no title attribute

h3. And fix the following:

Element is in tab order and does not have accessible text

!Screen Shot 2024-11-13 at 16.31.00 PM.png|width=782,height=406,alt="Screen Shot 2024-11-13 at 16.31.00 PM.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39570**



[created from NEXT-39570, comment 512384]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.0**

 This issue will be / was solved in Shopware version ***6.6.10.0***.



[created from NEXT-39570, comment 521035]
